.circleci/config.yml

```
@@ -274,7 +274,7 @@ jobs:
       - run:
           name: Build Notifications Panel
           command: |
-            NODE_ENV=production npm run build-notifications -- --output-path=$CIRCLE_ARTIFACTS/notifications-panel
+            NODE_ENV=production npx lerna run build --scope='@automattic/notifications' -- -- --output-path=$CIRCLE_ARTIFACTS/notifications-p
```

This was mistakenly truncated somehow which screws up the artifacts:

```suggestion
            NODE_ENV=production npx lerna run build --scope='@automattic/notifications' -- -- --output-path=$CIRCLE_ARTIFACTS/notifications-panel
```

Aside: anything built with `calypso-build` that's using `output-dir` to the artifact directory does not need an entry in `setup-results-and-artifacts` like `        "$CIRCLE_ARTIFACTS/notifications-panel"`, we could remove several lines as the directory is created on demand.